eql/parser.py

```diff
@@ -1040,7 +1053,8 @@ def subquery_by(self, node, num_values=None, position=None, close=None, allow_fo
         else:
             join_values = []
 
-        return NodeInfo(ast.SubqueryBy(query, [v.node for v in join_values], **kwargs), source=node), join_values
+        return NodeInfo(ast.SubqueryBy(query, [v.node for v in join_values], **kwargs),
+                        source=node), join_values, runs_count
```

Comment:
nit: since it's gonna be on two lines anyway, might as well use a variable
```suggestion
        node_info = NodeInfo(ast.SubqueryBy(query, [v.node for v in join_values], **kwargs), source=node)
        return node_info, join_values, runs_count
```